frontend/src/components/okr/two-tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import Tabs from 'react-bootstrap/Tabs';
+import Tab from 'react-bootstrap/Tab';
+import OrkItem from './workspace-item';
+import { useSelector } from 'react-redux';
+import { RootState } from 'common/types/types';
+import { IOkr, IObjective, IKeyResult } from './common/interfaces';
+
+interface PropTypes {
+  collection: IOkr[]
+}
+
+function ControlledTabs():React.ReactElement {
+    const [key, setKey] = useState('my-OKR');
+    const user = useSelector((state: RootState) => state.okr.User);
+    const okr = useSelector((state: RootState) => state.okr.OKR);
+    const myOkr = useSelector((state: RootState) => state.okr.OKR.filter(okr => okr.userId === user.id));
+    const objective = useSelector((state: RootState) => state.okr.Objective);
+    const keyResult = useSelector((state: RootState) => state.okr.KeyResult);
+
+    const OkrList: React.FC<PropTypes> = (props) => (
```

Comment:
1 component === 1 file

Also, use can get collection with useSelector and pass it to the OkrList.